config-examples/users.yaml

```diff
@@ -0,0 +1,4 @@
+users:
+  - name: default
+    password: default
+    perms: Read
```

Comment:
It is not a good approach to creat internal user managment. Could create a lots of security issue. It's better to authorize via SAML, OpenID Connect.

Reply:
bob is meant to be distributed. managing user in same way is not a nice.